client/src/App.css

```
@@ -23,6 +23,48 @@
   color: #61dafb;
 }
 
+.text-align-center {
+  text-align: center;
+  top: 50%;
+  line-height: 95px !important;
+}
+.left-margin-25 {
+  margin: 10px 0 0 25px;
```

same for this,

this CSS class says left margin but it also applies for the top as well.

Also, the name of what you're manipulating should be first like `margin-left-25`
```suggestion
  margin-left: 25px;
}
.margin-top-10 {
   margin-top: 10px;
```